src/tags/object/RichText/table.js

```diff
@@ -40,19 +54,51 @@ const renderTableValue = (val) => {
   }
 
   const itemClass = cn('richtext', { elem: 'table-item' });
+  const questionItemClass = cn('richtext', { elem: 'table-item', mod: { qa : 'question' } });
+  const mathItemClass = cn('richtext', { elem: 'table-item', mod: { context: 'math' } });
+  const questionMathItemClass = cn('richtext', { elem: 'table-item', mod: { qa : 'question', context: 'math' } });
+  let hasMath = false;
 
   const rowElems = conversations.map((conversation, index) => {
     const question = conversation[0];
     const answer = conversation[1];
+    const mathQuestion = extractMath(question);
+    const mathAnswer = extractMath(answer);
+    let mathQuestionComponent = null;
+    let mathAnswerComponent = null;
+
+    if (mathQuestion) {
+      mathQuestionComponent = <MathJax className={questionMathItemClass}>{MAJX_MARKER + mathQuestion + MAJX_MARKER}</MathJax>;
+      hasMath = true;
+    }
+    if (mathAnswer) {
+      mathAnswerComponent = <MathJax className={mathItemClass}>{MAJX_MARKER + mathAnswer + MAJX_MARKER}</MathJax>;
+      hasMath = true;
+    }
 
     return (
       <div key={`conversation-${index}`}>
-        <div className={itemClass}>{question}</div>
+        <div className={questionItemClass}>{question}</div>
+        {mathQuestionComponent}
         <div className={itemClass}>{answer}</div>
+        {mathAnswerComponent}
       </div>
     );
   });
 
+  if (hasMath) {
+    const mathJaxConfig = {
+      tex: {
+        inlineMath: [[MAJX_MARKER, MAJX_MARKER]],
+      },
+    };
+
+    return (
+      <MathJaxContext config={mathJaxConfig}>
```

Comment:
The `dynamic` flag seems to be for MathJax content changing.  However, it does not seems to trigger a new component to typeset (i.e. render into Latex), which is what we want.  I added a hacky way to drill a function into `ComponentDidMount`, which works good enough for now.

Reply:
https://github.com/fast-reflexes/better-react-mathjax#rendermode-pre--post--undefined
I was skimming the docs for the mathjax stuff since I'm totally unfamiliar with it, would setting the rendermode to post fix this? sounds like it causes the typeset to process on every render, which would be what we want here, right? I barely speak React, so maybe i'm way off here, or just not understanding the problem space.

Reply:
Ha picking this up after almost a year!

Thanks for the suggestions - I agree it sounds promising, but tried the `renderMode="pre"` together with combination of `<MathJax text="...."/>`, and it didnt work.  I didn't dig too deeply since the `ComponentDidMount` solution is working okay for now.